integ/scripts/bash/rfdk-integ-e2e.sh

```diff
@@ -90,11 +92,47 @@ for COMPONENT in **/cdk.json; do
     export ${COMPONENT_NAME}_START_TIME=$SECONDS
     if [[ "$COMPONENT_NAME" != _* ]]; then
         # Excecute the e2e test in the component's scripts directory
-        cd "$INTEG_ROOT/$COMPONENT_ROOT" && ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" || cleanup_on_failure
+        cd "$INTEG_ROOT/$COMPONENT_ROOT"
+        if [ "${RUN_TESTS_IN_PARALLEL}" = true ]; then
+            ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" || ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" --destroy-only &
+            export ${COMPONENT_NAME}_PID=$!
+            COMPONENTS+=(${COMPONENT_NAME})
+        else
+            ../common/scripts/bash/component_e2e.sh "$COMPONENT_NAME" || cleanup_on_failure
+        fi
     fi
     export ${COMPONENT_NAME}_FINISH_TIME=$SECONDS
 done
 
+if [ "${RUN_TESTS_IN_PARALLEL}" = true ]; then
+    while [ "${#COMPONENTS[@]}" -ne 0 ]; do
+        ACTIVE_COMPONENTS=()
+        for COMPONENT_NAME in ${COMPONENTS[@]}; do
+            PID=$(eval echo \"\$${COMPONENT_NAME}_PID\")
+            if ps -p "$PID" > /dev/null; then
+              ACTIVE_COMPONENTS+=(${COMPONENT_NAME})
+            else
+              echo "Test app $COMPONENT_NAME finished."
+              if [ -f "$INTEG_TEMP_DIR/${COMPONENT_NAME}_deploy.txt" ]; then
+                cat "$INTEG_TEMP_DIR/${COMPONENT_NAME}_deploy.txt"
+              fi
+              if [ -f "$INTEG_TEMP_DIR/${COMPONENT_NAME}.txt" ]; then
+                cat "$INTEG_TEMP_DIR/${COMPONENT_NAME}.txt"
+              fi
```

Comment:
This outputs stdout from this command in `integ/components/deadline/common/scripts/bash/deploy-utils.sh`:

```sh
yarn run test "$COMPONENT_NAME.test" --json --outputFile="$INTEG_TEMP_DIR/$COMPONENT_NAME.json" > "$INTEG_TEMP_DIR/${COMPONENT_NAME}.txt" 2>&1
```

When I tried running this code, it looks like there is no output that shows the test results. I believe jest is writing the JSON test report to `$INTEG_TEMP_DIR/$COMPONENT_NAME.json` and the original code would output this from `integ/scripts/bash/report-test-results.sh`. Strangely, I did not see the test results after running this myself. Maybe I'm missing something.

When you ran this, were you able to see the test result output?

Reply:
This is strange. I am looking on folder when I ran these tests locally and I can see 4 files for every test.
(`deadline_01_repository_deploy.txt, deadline_01_repository_destroy.txt, deadline_01_repository.json, deadline_01_repository.txt`) 
Have you run your test with `export RUN_TESTS_IN_PARALLEL=true`?

Reply:
I've re-ran this using your latest commit and no longer see the issue. Resolving